src/scanner/mod.rs

```diff
@@ -130,8 +137,33 @@ impl Scanner {
     ///
     /// Note: `self` must contain `self.ip`.
     async fn scan_socket(&self, socket: SocketAddr) -> io::Result<SocketAddr> {
-        let tries = self.tries.get();
+        if self.sudp {
+            // we try different wait times and packet types
+            // I looked for a crate to create these packets but couldn't find one
+            let waits = vec![0, 50, 100, 300];
```

Comment:
Suggestion: Make the times less even? A firewall would pick up on even timing. Ideally we'd use random times but it's slow and should be in a --quiet option (one for another PR)

Just doing things like 0, 51, 107, 313 etc would be good :)